kratos/sources/kratos_application.cpp

```diff
@@ -116,7 +116,16 @@ KratosApplication::KratosApplication(const std::string& ApplicationName)
       mpConditions(KratosComponents<Condition>::pGetComponents()),
       mpModelers(KratosComponents<Modeler>::pGetComponents()),
       mpRegisteredObjects(&(Serializer::GetRegisteredObjects())),
-      mpRegisteredObjectsName(&(Serializer::GetRegisteredObjectsName())) {}
+      mpRegisteredObjectsName(&(Serializer::GetRegisteredObjectsName())) {
+        
+        Registry::SetCurrentSource(mApplicationName);
```

Comment:
It is me or the spacing is inconsistent?

Reply:
is consistent, the inconsistent one is the initializer which is indented at 10 chars to have it in the same column as `mApplicationName`. We could change it in a different PR